MovieApp/ViewControllers/IntroViewController.swift

```diff
@@ -0,0 +1,299 @@
+            moveNextAction()
+        }
+    }
+
+    // MARK: - Private Methods
+
+    private func setupGestures() {
+        let swipeLeft = UISwipeGestureRecognizer(target: self, action: #selector(handleGesture))
+        swipeLeft.direction = .left
+        view.addGestureRecognizer(swipeLeft)
+        let swipeRight = UISwipeGestureRecognizer(target: self, action: #selector(handleGesture))
+        swipeRight.direction = .right
+        view.addGestureRecognizer(swipeRight)
+    }
+
+    private func setupView() {
+        view.backgroundColor = UIColor(named: Constants.ownDarkerColor)
+        view.addSubview(backView)
+        view.addSubview(darkView)
+        darkView.addSubview(moveNext)
+        darkView.addSubview(onboardLabel)
+        darkView.addSubview(onboardSubLabel)
+        darkView.addSubview(appTitleLabel)
+        darkView.addSubview(appSubTitleLabel)
+
+        for subView in view.subviews {
+            subView.translatesAutoresizingMaskIntoConstraints = false
+        }
+        for subView in darkView.subviews {
+            subView.translatesAutoresizingMaskIntoConstraints = false
+        }
+
+        moveNext.addTarget(self, action: #selector(moveNextAction), for: .touchUpInside)
+    }
+
+    private func moveBackAction() {
+        if currentPage >= 1 {
+            backLeadingAnchor.constant += view.frame.width
+            backTopAnchor.constant += (view.frame.width)
+
+            UIView.animate(
+                withDuration: 0.7,
+                delay: 0.1,
+                usingSpringWithDamping: 0.7,
+                initialSpringVelocity: 0.2,
+                options: .curveEaseIn
+            ) {
+                self.view.layoutIfNeeded()
+            } completion: { _ in
+                print("bitti")
+            }
+            currentPage -= 1
+
+            changeTextAction()
+        }
+    }
+
+    private func changeTextAction() {
+        UIView.animate(withDuration: 0.3) {
+            self.onboardLabel.alpha = 0
+            self.onboardSubLabel.alpha = 0
+            self.view.layoutIfNeeded()
+        } completion: { _ in
+            switch self.currentPage {
+            case 0:
+                self.onboardLabel.text = Constants.firstBoardText
+                self.onboardSubLabel.text = Constants.firstBoardSubText
+            case 1:
+                self.onboardLabel.text = Constants.secondBoardText
+                self.onboardSubLabel.text = Constants.secondBoardSubText
+            case 2:
+                self.onboardLabel.text = Constants.thirdBoardText
+                self.onboardSubLabel.text = Constants.thirdBoardSubText
+            default:
+                break
+            }
+            UIView.animate(withDuration: 0.3) {
+                self.onboardLabel.alpha = 1
+                self.onboardSubLabel.alpha = 1
+                self.view.layoutIfNeeded()
+            }
+        }
+    }
+
+    private func layoutSetup() {
```

Comment:
```suggestion
    private func layoutSetup() {
```
в этом методе каша нужно разнести по методам а то не читабельно